test-alerts.cpp

```diff
@@ -5,4 +5,50 @@
 
 TEST_CASE("infers the breach according to limits") {
   REQUIRE(inferBreach(12, 20, 30) == TOO_LOW);
+  REQUIRE(inferBreach(-12, 0, 35) == TOO_LOW);
+  REQUIRE(inferBreach(101,100,105) == NORMAL);
+  REQUIRE(inferBreach(25,20,30)==NORMAL);
+  REQUIRE(inferBreach(120,100,105) == TOO_HIGH);
+}
+
+TEST_CASE("Check for checkAndAlert functionality") {
+   BatteryCharacter batteryProp1 = {PASSIVE_COOLING, "ABCD"};
+  BatteryCharacter batteryProp2 = {HI_ACTIVE_COOLING, "BCDE"};
+  BatteryCharacter batteryProp3 = {MED_ACTIVE_COOLING, "PQRS"};
+  checkAndAlert(TO_CONTROLLER, batteryProp1, 20.0);
```

Comment:
how do we verify checkAndAlert has executed the way it is expected? There is no output nor verification of any state for ex. variable etc.

This seems like just a call. Please think of ways to verify the output or the state changes.